projects/sandbox/plots/plots/1D_param_hist.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+from bokeh.io import save
+from bokeh.layouts import gridplot
+from bokeh.plotting import figure
+from typeo import scriptify
+
+from aframe.analysis.ledger.events import RecoveredInjectionSet
+from aframe.analysis.ledger.injections import InjectionParameterSet
+from aframe.logging import configure_logging
+
+
+@scriptify
+def main(
+    foreground_file: Path,
+    rejected_params: Path,
+    output_fname: Path,
+    log_file: Optional[Path] = None,
+    verbose: bool = False,
+):
+    configure_logging(log_file, verbose)
+
+    logging.info("Reading in inference outputs")
+
+    foreground = RecoveredInjectionSet.read(foreground_file)
+    rejected = InjectionParameterSet.read(rejected_params)
+    all_params = {
+        k: np.concatenate(((foreground[k], rejected[k])))
+        for k in rejected.keys()
+    }
+
+    labels = [
```

Comment:
You might want to check whether this list of labels is equivalent to `list(rejected.keys())`. Could save you some space. If not, then this is fine as-is